public/static/app/perfdiscovery/PerformanceDiscoveryCtrl.js

```diff
@@ -85,23 +85,48 @@ mciModule.controller('PerformanceDiscoveryCtrl', function(
 
   let oldFromVersion, oldToVersion;
 
+  // Convert the name to a revision that can be used to query atlas.
+  const nameToRevision = (name) => {
+    if(name) {
+      const parts = name.split(/[-_]/);
+      name = parts[parts.length -1];
```

Comment:
[nit] Could you add a space between '-' and '1'?

Reply:
done